haiku-apps/wolle/wolle-0.1~git.recipe

```diff
@@ -0,0 +1,45 @@
+srcGitRev="be9165f52bee14d3cd1d19600ce6cfa44f217df4"
+SUMMARY="A screensaver displaying  woollen string"
```

Comment:
Just a cosmetic changes: Remove one of the space between "displaying" and "woollen".